backend/tests/v5/helper/services.js

```diff
@@ -68,31 +68,28 @@ db.createModel = (teamspace, _id, name, props) => {
 	return DbHandler.insertOne(teamspace, 'settings', settings);
 };
 
-db.createRevision = (teamspace, modelId, { _id, author, timestamp, tag }) => {
-	const rev = {
-		_id: stringToUUID(_id),
-		author,
-		timestamp,
-		tag,
-	};
-	return DbHandler.insertOne(teamspace, `${modelId}.history`, rev);
+db.createRevision = (teamspace, modelId, revision) => {
+	const formattedRevision = { ...revision, _id: stringToUUID(revision._id) };
+	return DbHandler.insertOne(teamspace, `${modelId}.history`, formattedRevision);
 };
 
 ServiceHelper.generateUUIDString = () => uuidToString(generateUUID());
 ServiceHelper.generateRandomString = () => Crypto.randomBytes(15).toString('hex');
+ServiceHelper.generateRandomDate = (start, end) => new Date(start.getTime() + Math.random()
+	* (end.getTime() - start.getTime()));
 
 ServiceHelper.generateUserCredentials = () => ({
 	user: ServiceHelper.generateRandomString(),
 	password: ServiceHelper.generateRandomString(),
 	apiKey: ServiceHelper.generateRandomString(),
 });
 
-ServiceHelper.generateRevisionEntry = () => ({
+ServiceHelper.generateRevisionEntry = (isVoid = false) => ({
 	_id: ServiceHelper.generateUUIDString(),
 	tag: ServiceHelper.generateRandomString(),
-	timestamp: new Date(),
 	author: ServiceHelper.generateRandomString(),
-
+	timestamp: ServiceHelper.generateRandomDate(new Date(2018, 1, 1), new Date()),
+	void: isVoid,
```

Comment:
cast it to boolean incase somsone pass in a string 

```suggestion
	void: !!isVoid,
```